app/frontends/ogc-coverages/util/parameter-parsing.ts

```diff
@@ -160,6 +160,44 @@ function _getDimensionName(value: string): string {
   }
 }
 
+/**
+ * Parses the provided point parameters and ensures they are valid, throwing an error message
+ * if not
+ *
+ * @param values - An array of all the specified point parameters from the request
+ * @param dimConfig - A mapping of dimension names to min, max, and data type values,
+ *   see `dimensionInfo` (the default value) in this file.  Usually should not be specified,
+ *   except for testing.
+ * @returns An array with two elements corresponding to [longitude, latitude]
+ * @throws ParameterParseError - if a subset parameter cannot be parsed, has unrecognized
+ *   axis names, or is otherwise invalid
+ */
+export function parsePointParam(
+  values: string,
+  dimConfig: DimensionConfig = dimensionConfig,
+): number[] {
+  let results;
+  let coordinate, coordinates;
+  if (values !== undefined){
+    coordinates = values;
+    if ( coordinates.length !== 2 )
```

Comment:
is this correct? `values` is a string - should it be length === 2?

Reply:
Good catch ... actually `values` is supported to be an array.
Fixed.